app/assets/javascripts/steps.js

```diff
@@ -107,6 +108,34 @@
       });
     },
 
+    bindOverviewTableFilter: function() {
```

Comment:
Something doing this much work should really be tested 🤔

Reply:
We've talked in person about adding a card to the backlog to set-up Javascript tests, so happy to resolve this.